### PR TITLE
Add Corsican localization

### PR DESCRIPTION
**Pull request by alixrossi**

### CODE REVIEW

File: config/locales/activerecord.co.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+en:
```
Comment (Gargron) [inline config/locales/activerecord.co.yml]: co

**Comment by palindromordnilap** [comment]
Okay, it seems I need to run `i18n-tasks normalize` on the files. Unfortunately I can't do that on the one computer I have access to right now. Oh well.

**Comment by Gargron** [comment]
You don't have to close the PRs every time... Maybe someone will come along and run i18n-tasks normalize for you...

**Comment by palindromordnilap** [comment]
I'm closing it because I'm gonna make a bunch of commits and pushes and I'm not gonna spam that PR with them until I'm done.

**Comment by nightpool** [comment]
it's better to keep the PR open, so it only takes up 1 place in the list.

**Comment by palindromordnilap** [comment]
I can actually reopen it without making a new one, I just didn't notice that the first time.

**Pull request closed without merging**